apps/console/src/features/users/components/guests/pages/guest-users-list.tsx

```diff
@@ -305,19 +304,18 @@ export const GuestUsersList: FunctionComponent<GuestUsersListInterface> = (
                 id: "email",
                 key: 0,
                 render: (invite: UserInviteInterface) => (
-                    <Header as="h6" image>
-                        <AppAvatar
-                            image={ (
-                                <AnimatedAvatar
-                                    name={ invite.email }
-                                    size="mini"
-                                    data-componentid={ `${ componentId }-item-image-inner` }
-                                />
-                            ) }
+                    <Header 
+                        image
+                        as="h6" 
+                        className="header-with-icon"
+                        data-componentid={ `${ componentId }-item-heading` }
```

Comment:
Did we intentionally change the data component-id? Let's check if this affects the test (just in case)

Reply:
Fixed `data-componetid`'s.